index.ts

```diff
@@ -0,0 +1,11 @@
+import * as fs from "fs";
```

Comment:
it's a good practice to only import the methods that you're using - especially if you are using only one or few of them
```suggestion
import { writeFileSync } from "fs"
```